examples/styled/README.md

```diff
@@ -13,14 +13,13 @@ import { ImageFocalPoint } from '@lemoncode/react-image-focal-point';
 const App = () => {
   return (
     <ImageFocalPoint
-       src="your-image-src"
-       onChange={(newFocalPoint) => {
-         // Whatever you want to do when the user clicks on the image
-       }}
-     />
+      src="your-image-src"
+      onChange={newFocalPoint => {
+        // Whatever you want to do when the user clicks on the image
```

Comment:
Add here your code to do whatever you want to when the user drags on the focal point